lldb/examples/synthetic/gnu_libstdcpp.py

```diff
@@ -68,6 +68,11 @@ def extract_type(self):
         template_arg_num = type.GetNumberOfTemplateArguments() - 1
         allocator_type = type.GetTemplateArgumentType(template_arg_num)
         data_type = allocator_type.GetTemplateArgumentType(0)
+        if data_type.IsValid():
+            return data_type
```

Comment:
Can we just remove this codepath and use the same codepath for Clang and GCC? That way it's also going to be tested on CI.